File: CPPScript/Функции/Функция создания папки видео.cpp
///////////////////////////////////////////////////////////////////////////////
// --- Создание папки видео ---------------------------------------------------
THmsScriptMediaItem CreateFolder(string sName, string sLink, string sImg='') {
  THmsScriptMediaItem Item = PodcastItem.AddFolder(sLink); // Создаём папку с указанной ссылкой
  Item[mpiTitle     ] = sName; // Присваиваем наименование
  Item[mpiThumbnail ] = sImg;  // Картинка
  Item[mpiCreateDate] = DateTimeToStr(IncTime(gStart, 0, 0, -gnTotalItems, 0));
  gnTotalItems++;             // Увеличиваем счетчик созданных элементов
  return Item;                // Возвращаем созданный объект
}

File: PascalScript/Функции/Функция создания папки видео.pas
///////////////////////////////////////////////////////////////////////////////
// --- Создание папки видео ---------------------------------------------------
Function CreateFolder(aName, aLink: String; aImg: String=''): THmsScriptMediaItem;
Begin
  Result := PodcastItem.AddFolder(sLink); // Создаём папку с указанной ссылкой
  Result[mpiTitle     ] := sName; // Присваиваем наименование
  Result[mpiThumbnail ] := sImg;  // Картинка
  Result[mpiCreateDate] := DateTimeToStr(IncTime(gStart, 0, 0, -gnTotalItems, 0));
  Inc(gnTotalItems);              // Увеличиваем счетчик созданных элементов
End;
